lib/common/socket/evloop/epoll.c.h

```diff
@@ -82,6 +82,12 @@ static int update_status(struct st_h2o_evloop_epoll_t *loop)
                 } else {
                     assert(ev.events != 0);
                     op = EPOLL_CTL_ADD;
+#if defined(EPOLLEXCLUSIVE)
+                    /* EPOLLEXCLUSIVE: since Linux 4.5 glibc 2.24. */
+                    if ((sock->_flags & H2O_SOCKET_FLAG_DONT_READ) != 0) {
```

Comment:
Would you mind creating a different socket flag (possibly `H2O_SOCKET_FLAG_POLL_EXCLUSIVE`) that instructs the use of `EPOLLEXCLUSIVE` (or an option alike on other platforms), and let the callers of `h2o_evloop_socket_create` creating listening sockets set the socket flag?

I ask this because use of `H2O_SOCKET_FLAG_DONT_READ` does not necessary imply the necessity of setting `EPOLLEXCLUSIVE`. I want to be cautious on that, because we are starting to use evloop for handling UDP packets as well.

Reply:
@kazuho Thanks for your comments.
It's reasonable adding `H2O_SOCKET_FLAG_POLL_EXCLUSIVE` and let the caller decide whether to enable or not.